.github/labels.md

```diff
@@ -0,0 +1,249 @@
+the implementation.  When applying this Label reference the new Pull Request
+number in the old one.
+
+#### Label `status:help wanted`
+
+??? I don't think we really need that.  At least I didn't see any Issue were it
+caused someone required to help to magically appear.  This can be handled better
+with `@`-mentions if someones or some groups special knowledge is required.
+
+#### Label `status:invalid`
+
+The Issue doesn't classified as formal valid Issue report or applicable Pull
+Request.  Reasons for this can be manifold:
+
+* The Issue isn't describing a bug but intended behaviour.
+* The Issue opposes the design goals of the game.
+* The issue contains SPAM.
+* The description is too sparse to make sense out of the intent of the Issue
+  and the Reporter is unresponsive or unwilling to give further details.
+* The Feature implemented wasn't discussed previously with the team.
+* The Pull Request is a merge salad à la Chef and the developer is unable or
+  unwilling to fix this.
+* The commits contain intentional written game-braking or offending changes and
+  the developer is unwilling to change those.
+* Multiple Pull Requests compete to address the same topic, but were not chosen
+  to be merged.
+
+Issues labeled as `status:invalid` shouldn't be merged and should be closed.
+
+#### Label `status:merged`
+
+All relevant commits of this Pull Request were merged into the master
+development branch.  This can either happen by merging, cherry-picking or
+rebasing and pushing onto master.  Close the Issue if GitHub didn't close
+it automatically.
+
+#### Label `status:resolved`
+
+The Issue was resolved satisfactorily, either by answering properly or fixing
+the underlying bug.  Close the Issue.
+
+#### Label `status:testing requested`
+
+The Implementation is ready for testing but can't be tested sufficiently by the
+developer and they request testing by other developers.  Use this in case of
+platform specific code or bug fixing when the Bug is not reproducible on the
+developer's machine.
+
+#### Label `status:won't fix`
+
+The Issue describes a bug but won't be fixed by the developers because of the
+implementation or maintainance effort required.  For example a valid Issue for
+an old and unsupported version is written it would be labeled as
+`status:won't fix`.
+
+Add a comment explaining why the Issue won't be fixed and close the Issue.  This
+is __NOT__ a way to brush away user concerns or bad user experience so rethink
+if the Issue can be avoided by redesigning the application part in question in
+another way that the Issue may suggest.
+
+#### Label `status:workaround`
+
+??? I don't know if we really need this.  I intended it for noting a potential
+available workaround for a Issue but does it matter that much that we can't
+write a comment for it and instead need a label?
+
+#### Label `status:work in progress`
+
+The Pull Request contains some implementation but isn't ready for a review.
```

Comment:
I wouldn't say not ready for *review* (why creating a PR in that case anyway?), but not ready for *merge*. Tagging a PR as "work in progress" tells the other devs that the author already wants some feedback, even testing of what is already there, but that the implementation isn't complete yet, and there still needs work to be done.

It basically says: don't merge this yet.